src/index.ts

```diff
@@ -0,0 +1,27 @@
+import express, { Express, Request, Response } from 'express';
+import * as dotenv from 'dotenv';
+import { initDb } from './db';
+import usersRouter from './Routers/usersRouter';
+import bodyParser from 'body-parser';
+
+dotenv.config();
+
+const app: Express = express();
+const port = process.env.PORT || 3000;
+
```

Comment:
First of all, I suggest you use cors middleware, for all of your responses.
You can write app.use(cors()). And you need to install cors as a dep. Because Postman doesn't requires Access-Control-Allow-Origin header in server responses, but browser does. And if you will try to make a request to your backend, you will get CORS error in browser console.
Second, you need to allow EXPRESS to handle errors by itself. In every method, such app.get, etc, express pass next method, which you can invoke in case of error.
And in index.ts you can write handler for errors.

Reply:
add app.use(cors()) and express eror handler